waltz-test/src/main/python/waltz_ducktape/services/waltz_storage.py

```diff
@@ -5,31 +5,31 @@ class WaltzStorageService(BaseWaltzService):
     """
     WaltzStorageService is the service class for Waltz Storage.
     """
-    def __init__(self, context, cluster_spec, zk, cluster_root, cluster_num_partitions, cluster_key, port, admin_port,
-                 jetty_port, lib_dir, data_dir, config_file_dir):
+    def __init__(self, context, cluster_spec, zk, cluster_root, port, admin_port, jetty_port, lib_dir, data_dir,
+                 config_file_dir, ssl_configs):
         """
         Construct a new 'WaltzStorageService' object.
 
         :param context: The test context
         :param cluster_spec: The cluster specifics
         :param zk: Zookeeper url
         :param cluster_root: The cluster root
-        :param cluster_num_partitions: Number of partitions in cluster
-        :param cluster_key: The cluster key
         :param port: The service port
         :param admin_port: The admin port
         :param jetty_port: The jetty port
         :param lib_dir: The library directory
         :param data_dir: The data directory
         :param config_file_dir: The directory of configuration file path
+        :param ssl_configs: A dict of ssl related configurations
         """
         super(WaltzStorageService, self).__init__(context, cluster_spec=cluster_spec, zk=zk, cluster_root=cluster_root, port=port, \
                                                   lib_dir=lib_dir, config_file_dir=config_file_dir)
-        self.cluster_num_partitions = cluster_num_partitions
-        self.cluster_key = cluster_key
```

Comment:
Wonder why do we want to remove the cluster_key here? Is it because now we have `cluster_root` and could fetch those info from it?

Reply:
Thats because WaltzStorage fetches this information from the StoreMetadata (via ZooKeeper Connect String). This change was made sometime end of last year. This code was not updated after that.

Reply:
I have updated the hardcoded log4j path. Thanks for pointing it out. 👍